dist/ramda.js

```diff
@@ -6200,10 +6198,6 @@
     };
 
     var R = {
-        F: F,
-        I: I,
-        T: T,
-        __: __,
```

Comment:
Yes, busy last night.  I wanted to add the comments to address Mike's note below, and then will rebase and hope it works.  I still don't actually understand precisely why it failed, but I do understand what the fix in #824 does.  If that fixes it, great!